doc/api/stream.md

```diff
@@ -1221,23 +1511,20 @@ transform.prototype._transform = function (data, encoding, callback) {
 };
 ```
 
-This method is prefixed with an underscore because it is internal to
-the class that defines it, and should not be called directly by user
-programs. However, you **are** expected to override this method in
-your own extension classes.
+The `tranform.\_transform()` method is prefixed with an underscore because it is
+internal to the class that defines it, and should never be called directly by
+user programs.
 
-#### Example: `SimpleProtocol` parser v2
+#### An Example Transform Stream: `SimpleProtocol` parser, Version 2
 
 The example [here](#stream_example_simpleprotocol_v1_sub_optimal) of a simple
-protocol parser can be implemented simply by using the higher level
-[Transform][] stream class, similar to the `parseHeader` and `SimpleProtocol
-v1` examples.
+protocol parser can be implemented more efficiently as a [Transform][] class.
 
 In this example, rather than providing the input as an argument, it
 would be piped into the parser, which is a more idiomatic Node.js stream
 approach.
 
-```javascript
+```js
```

Comment:
We should probably nix this example and replace it with an example of duplexing streams (which would be more useful in practice than a header parser.)